packages/node/src/sessionInfo/SessionInfoManager.ts

```diff
@@ -111,26 +112,35 @@ export class SessionInfoManager implements ISessionInfoManager {
     throw new Error("Not Implemented");
   }
 
-  async get(sessionId: string): Promise<ISessionInfo | undefined> {
+  async get(
+    sessionId: string
+  ): Promise<(ISessionInfo & ISessionInternalInfo) | undefined> {
     const webId = await this.storageUtility.getForUser(sessionId, "webId");
     const isLoggedIn = await this.storageUtility.getForUser(
       sessionId,
       "isLoggedIn"
     );
+    const refreshToken = await this.storageUtility.getForUser(
+      sessionId,
+      "refreshToken"
+    );
+    const issuer = await this.storageUtility.getForUser(sessionId, "issuer");
```

Comment:
Just a thought (for later), but would it make sense to use vocab terms (i.e., IRIs) for the 'keys' that appear here (and for all the localStorage keys we use throughout our code), i.e., `webId`, `refreshToken` and `issuer`? It would make the keys in storage a few bytes longer, but would make them completely globally unique, and more importantly, self-descriptive (i.e., dereference the IRI to see the `rdfs:comments` (in multiple languages!), and any other meta-data related to that term)? Would also mean we'd get rid of all these nasty magic strings...
```suggestion
    const webId = await this.storageUtility.getForUser(sessionId, "webId");
    const isLoggedIn = await this.storageUtility.getForUser(
      sessionId,
      "isLoggedIn"
    );
    const refreshToken = await this.storageUtility.getForUser(
      sessionId,
      "refreshToken"
    );
    const issuer = await this.storageUtility.getForUser(sessionId, "issuer");
```